com.stansassets.build/Editor/Core/IBuildExecutorListener.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace StansAssets.Build.Editor
+{
+    /// <summary>
+    /// Single listener which runs when build started.
+    /// Will be called before run all steps and tasks
+    /// </summary>
+    public interface IBuildExecutorListener
+    {
+        /// <summary>
+        /// Checking if include this listener to build
```

Comment:
Probably something like: "Indicates is listener active or not while registration flow. It allows toggle build steps/tasks registration without re-compile."